objectRecognitionSource/README.md

```diff
@@ -279,6 +279,95 @@ Options available for all Queries (not prepended by anything) are:
     instead of only the objects recognized. Note that the data will be the sole occupant of a 1-element array when
     received, instead of being immediately available as a JSON object. This is because Query results are mandated
     to be arrays. Default is false.
+    
+Unique query capabilities available for the YOLO Processor are:
+
+*   **NOTE:** All YOLO Processor queries must have an "operation" parameter specified. If this parameter is not specified, it 
+will be set to the default value, "processNextFrame".
+
+*   **Upload images to VANTIQ:**
+    *   The user can specify an image, or multiple images to be uploaded to VANTIQ as a document. The images will be those 
+    that are saved in the output directory, which is defined in the source configuration.
+    *   Parameters:
+        *   "operation": Required. Must be set to "upload".
+        *   *Only one of the following two values *MUST* be specified, or a query error will be returned. If both values are 
```

Comment:
This section is inconsistent:  first, it says "only one" otherwise an error. Then it says, if both, we'll pick one.  We need to choose between these behaviors.

Reply:
Perhaps it's trying to say that _at least one_ must be there.  But we're really saying that _exactly one_ should be there.  But if you say both, we'll pick the more specific?

Reply:
I think I will change this to "at least one". Right now, if you specify both then it will pick the more specific one, which is imageName.

Reply:
We should probably say that we want exactly one.  But say it like "you should specify exactly one of the following..."  then we can give the behavior if you screw up.